### ISSUE TITLE
[community] Bi-weekly community developers meeting

### ISSUE THREAD

**Issue by jhamman** [pydata/xarray#4001]
Hello Xarray Community and @pydata/xarray, 

Starting next week, we will be hosting a bi-weekly 30-minute community/developers meeting. The goal of this meeting is to help coordinate Xarray development efforts and better connect the user/developer community. 

### When
Every other Wednesday at 8:30a PT (11:30a ET, 3:30p UTC) beginning April 29th, 2020.

Calendar options:
- [Google Calendar](https://calendar.google.com/calendar/embed?src=ucar.edu_2gjd5fuugcj4ol6ij7knj8krn8%40group.calendar.google.com&ctz=America%2FLos_Angeles)
- [Ical format](https://calendar.google.com/calendar/ical/ucar.edu_2gjd5fuugcj4ol6ij7knj8krn8%40group.calendar.google.com/public/basic.ics)

### Where
TBD. Check back soon for zoom/whereby link...

### Rolling agenda and meeting notes
We'll keep a rolling agenda and set of meeting notes [here](https://hackmd.io/@U4W-olO3TX-hc-cvbjNe4A/xarray-dev-meeting/edit).

**Comment by jhamman** [comment]
Zoom is not working for us today, we'll meet here instead: https://carbonplan.whereby.com/xarray

**Comment by mathause** [comment]
@keewis I think this might be 17:30 CET again next time when the daylight saving time ends in the US

**Comment by keewis** [comment]
I guess that depends on which time we want to keep constant, PT or UTC. Any thoughts?

**Comment by shoyer** [comment]
Next week I think Europe and the US will be back in sync with daylight saving time. So both the PT and CET times will be good again?

**Comment by keewis** [comment]
okay, let's go with that. I'll update the first post, then.

**Comment by max-sixty** [comment]
You're right to remove the CET & UTC... the Google Cal should update for the relative timezones if people use that.

**Comment by raybellwaves** [comment]
Any interest in adding this to docs similar to https://docs.dask.org/en/latest/support.html
I'm also fine if it's purpose is just for core devs

**Comment by max-sixty** [comment]
For sure @raybellwaves !

**Comment by TomNicholas** [comment]
Is someone able to let us in to the zoom meeting room today?

**Comment by jhamman** [comment]
Something seems to have changed in our regular Zoom room. For this week, let's use [this Zoom link](https://us02web.zoom.us/j/6056181838?pwd=clRRSVprQ0p2RnZLS2VUbmJlM1QvQT09).

**Comment by jhamman** [comment]
@pydata/xarray  and others - we're going to cancel the next regularly scheduled meeting (Dec. 22, 2021) so everyone can take a break. We'll be back Jan. 5, 2022.

**Comment by JessicaS11** [comment]
Just wanted to let you know that the hackmd notes link in the google calendar leads to a 404 (the one at the top of this issue works though). Google also says the calendar url is invalid if I try to add it to my calendar (even though it works in a browser).

**Comment by dcherian** [comment]
I updated the hackMD link but it requires sign-in to view and I can't change that (@rabernat )

**Comment by shoyer** [comment]
Can we delete the "Flexible indexes" meeting? It doesn't happen anymore.
